src/Subscription.php

```diff
@@ -612,6 +612,28 @@ public function getCurrencyAttribute()
         return optional($this->plan())->amount()->getCurrency()->getCode();
     }
 
+    /**
+     * Gets the amount to be refunded for the subscription's unused time.
```

Comment:
"reimbursed" instead of "refunded"

Reply:
Also good to add a note here on the resulting value range (0 to positive probably)

Reply:
@sandervanhooft it's a negative value. So the range is from negative x to 0